src/utils/common/nixl_duration.cpp

```
@@ -0,0 +1,118 @@
+
+    return cal;
+}
+
+} // namespace
+
+const fastClockCal g_fastClockCal = calibrate();
+
+} // namespace detail
+
+[[nodiscard]] bool
+fastClockUsesHwCounter() {
```

Cppcheck (2.21.0)

[style] The function 'fastClockUsesHwCounter' is never used.

(unusedFunction)